src/main/java/racingcar/repository/RacingCarRepository.java

```diff
@@ -0,0 +1,14 @@
+package racingcar.repository;
+
+import java.util.List;
+
+import racingcar.repository.entity.Record;
+import racingcar.service.RacingCarResult;
+
+public interface RacingCarRepository {
+
+    List<Record> findAllRecords();
```

Comment:
```suggestion
    List<Record> findAll();
```

리턴 타입이 Record인데 메서드명에 또 쓸 필요가 있나요?